### PR TITLE
chore(test): skip console messages containing 'deprecated'

### PR DESCRIPTION
**Pull request by matthewmayer**
running `pnpm run test` logs an enormous number of deprecation warnings. This makes it hard to review the output locally and in CI. 
https://github.com/vitest-dev/vitest/issues/1700 shows a way to filter out specific logs, in this case anything containing "deprecated".

### CODE REVIEW

**Comment by ST-DDT** [comment]
Please fix the lint errors.

**Review by ejcheng — DISMISSED** [review]

**Review by ST-DDT — DISMISSED** [review]
This seems to improve the execution speed in CI (3.5mins -> 2.5mins), but it doesn't have that much of an impact for me locally.

The logs have been reduced from almost 30k logs to ~200 which is good, but I'm not sure this a good long term solution.

File: vite.config.ts [hunk 1]
```diff
@@ -9,5 +9,8 @@ export default defineConfig({
       reporter: ['clover', 'cobertura', 'lcov', 'text'],
       include: ['src'],
     },
+    onConsoleLog(log) {
+      if (log.includes('deprecated')) return false;
+    },
```
Comment (Shinigami92) [inline vite.config.ts]:
```suggestion
    onConsoleLog(log, type) {
      if (
        type === 'stderr' &&
        log.includes('[@faker-js/faker]:') &&
        log.includes('deprecated')
      ) {
        return false;
      }
    },
```

**Review by Shinigami92 — COMMENTED** [review]
Tested before and after my changes
Both still don't really work better on my machine
The tests are absolutely bloated and run into timeouts

**Comment by matthewmayer** [comment]
I don't know enough about how the tests works to figure out why they are so slow, however this change at least makes it possible to read the log output on GitHub.com easily.

**Review by Shinigami92 — APPROVED** [review]

**Comment by Shinigami92** [comment]
> [...], however this change at least makes it possible to read the log output on GitHub.com easily.

This is indeed a huge benefit 👌

**Review by ejcheng — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**